assets/app/scripts/directives/util.js

```diff
@@ -94,4 +94,22 @@ angular.module('openshiftConsole')
       },
       templateUrl: 'views/directives/_custom-icon.html'
     };
+  })
+  .directive('bottomOfWindow', function() {
+    return {
+      restrict:'A',
+      link: function(scope, element) {
+        function resized() {
+          var height = window.innerHeight - element[0].getBoundingClientRect().top;
+          element.css('height', (height - 10) + "px");
+        }
+
+        window.addEventListener('resize', resized);
```

Comment:
we typically use $( window ).resize(resize) instead of directly calling window.addEventListener